Dockerfile

```
@@ -17,13 +17,13 @@ RUN groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
 
 WORKDIR /usr/src/app
 COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
-RUN npm install --production --silent && mv node_modules ../
-RUN npm install -g ts-node typescript nodemon
+RUN yarn --production --silent && mv node_modules ../
+RUN yarn global ts-node typescript nodemon
```

There is a compile error at this command
I think you forgot the add prefix
```suggestion
RUN yarn global add ts-node typescript nodemon
```